magtogoek/adcp/loader.py

```diff
@@ -145,6 +152,8 @@ def load_adcp_binary(
         data = RtiReader(filenames=filenames).read(
             start_index=leading_index, stop_index=trailing_index
         )
+        if magnetic_declination_setting:
```

Comment:
For an easier understanding of the code, I would avoid using broad statement like `if variable:`, since False, None, ''. (), [], {}, 0 .. return False. Specifying the False condition also will make the code more rigid, thus less likely to have unnoticed soft error. In that case, it would be `if magnetic_declination_setting is not None:`, now we know that we are either expecting a None or numerical value.